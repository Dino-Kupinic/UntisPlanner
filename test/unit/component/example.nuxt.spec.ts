import {it, expect} from "vitest"
import {mountSuspended} from "@nuxt/test-utils/runtime"
import {Example} from "#components"

it("can mount some component", async () => {
  const component = await mountSuspended(Example)
  expect(component.html()).toMatchSnapshot("<p>Example</p>")
})