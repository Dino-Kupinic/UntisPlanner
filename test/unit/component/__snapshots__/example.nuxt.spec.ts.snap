// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Example Component > can mount some component > <p>Example</p> 1`] = `"<p>Example</p>"`;

exports[`Example Component > can mount some component 1`] = `"<p>Example</p>"`;

exports[`can mount some component > <p>Example</p> 1`] = `"<p>Example</p>"`;
